language_service/src/hover/tests.rs

```diff
@@ -831,3 +844,40 @@ fn callable_empty_summary() {
         "#]],
     );
 }
+
+#[test]
+fn callable_param_doc() {
+    check(
+        indoc! {r#"
+        namespace Test {
+
+            /// Doc string
+            /// # Summary
+            /// This is the summary
+            /// # Input
+            /// Input string
+            /// ## x
+            /// Doc string for `x`
+            /// ### Note
+            /// note for `x`
+            /// ## other
+            /// Doc string for `other`
+            /// # Last
+            /// Last string
+            operation Foo(x: Int) : Unit {
+                let y = ◉↘x◉;
+            }
+        }
+    "#},
+        &expect![[r#"
+            param of `Foo`
+            ```qsharp
+            x: Int
+            ```
+            ---
+            Doc string for `x`
+            ### Note
+            note for `x`
```

Comment:
Same point as my comment about line breaks above. I suggest you check this output with a couple of different markdown renderers. I would expect double line breaks between each block here for proper formatting, but I may be wrong.

Reply:
I've looked at it on both the playground and in VS Code, and it looks fine in both to me. What other renderers would you recommend I check it against?

Reply:
Oh that sounds good then. I think my previous markdown mistakes made me paranoid. I just pasted it into this GitHub comment and checked "Preview" too and I saw no problem. Thanks for checking!